src/client/BuilderClient.ts

```diff
@@ -1,50 +1,34 @@
-import axios, { AxiosInstance } from 'axios'
 import FormData from 'form-data'
 import { Authenticator, AuthIdentity } from 'dcl-crypto'
 import { Content } from '../content/types'
 import { RemoteItem, LocalItem } from '../item/types'
 import { ClientError } from './BuilderClient.errors'
 import { ServerResponse } from './types'
+import crossFetch from 'cross-fetch'
 
 export class BuilderClient {
-  private axios: AxiosInstance
+  private fetch: (url: string, init?: RequestInit) => Promise<Response>
   private readonly AUTH_CHAIN_HEADER_PREFIX = 'x-identity-auth-chain-'
 
   constructor(
     url: string,
     private identity: AuthIdentity,
-    private address: string
+    private address: string,
+    externalFetch: typeof fetch = crossFetch
```

Comment:
is this `typeof fetch` using `global` to get the typing? Maybe we want to explicitly import it? I might be missing something!

Reply:
The thing is that you can't import the dom's fetch type. We can either use the crossFetch type or leave it as is so the compiled types match the ones of the global one.